### PR TITLE
prepare the move to prometheusv2 - start with the old values and adju…

### PR DESCRIPTION
**Pull request by thgrs**
…sting in a later commit

### CODE REVIEW

**Comment by auhlig** [comment]
starts a 2nd prometheus in version v2.5.0 with configmap reload sidecar. needs to write to a separate folder than prom v1.x as datamodel changed. 
not sure 100GiB is sufficient for both prometheis tsdb's

File: openstack/maia/templates/prometheus-deployment.yaml [hunk 1]
```diff
@@ -41,6 +41,38 @@ spec:
               mountPath: /prometheus
             - name: config
               mountPath: /etc/prometheus
+{{- if .Values.prometheusv2.enabled }}
+        - name: prometheusv2
+          image: "{{ .Values.prometheusv2.image.repository }}:{{ .Values.prometheusv2.image.tag }}"
+          args:
+            - --config.file=/etc/prometheus/prometheusv2.yaml
+            - --storage.tsdb.path=/prometheusv2
```
Comment (thgrs) [inline openstack/maia/templates/prometheus-deployment.yaml]: @auhlig i would suggest to use /prometheus/v2 here - this way we do not conflict with the v1 files, which we will delete later

File: openstack/maia/templates/prometheus-deployment.yaml [hunk 2]
```diff
@@ -41,6 +41,38 @@ spec:
               mountPath: /prometheus
             - name: config
               mountPath: /etc/prometheus
+{{- if .Values.prometheusv2.enabled }}
+        - name: prometheusv2
+          image: "{{ .Values.prometheusv2.image.repository }}:{{ .Values.prometheusv2.image.tag }}"
+          args:
+            - --config.file=/etc/prometheus/prometheusv2.yaml
```
Comment (thgrs) [inline openstack/maia/templates/prometheus-deployment.yaml]: @auhlig i think we would have to provide this files then as well - would it have the same content as the v1 one?

Comment (thgrs) [reply openstack/maia/templates/prometheus-deployment.yaml]: discussed with arno: same config file, so we will use prometheus.yaml

File: openstack/maia/templates/prometheus-deployment.yaml (hunk 1 above)
Comment (thgrs) [reply openstack/maia/templates/prometheus-deployment.yaml]: v2 subdir is ok

**Pull request merged**